### ISSUE TITLE
Sculpting, Facesets menu - add hotkey only Face Sets Visibility operators

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#4173]
Hotkey only operators is no go. They need a menu entry.

I could have sworn that we had these in the UI at one point. But Bfa 3.5 was already without it, and i didn't want to dig deeper. Task remains, add them into the menu. 

![missing](https://github.com/Bforartists/Bforartists/assets/13919702/d332280c-9807-4c15-820c-5105529076a0)

**Comment by ReinerBforartists** [comment]
Can of worms again. The operators does not immediately update. Even worse, you have to click at the last operator popup to perform the show hide. Lemme have a look if it is reproduceable in Blender.

**Comment by ReinerBforartists** [comment]
Blender works just fine. The problem is not only our new menu, but the hotkeys already refuses to work. No idea what the problem really is.

With the menu items i have a further problem. It tells me it is waiting for an input. Very odd.

![inputpending](https://github.com/Bforartists/Bforartists/assets/13919702/a2e75467-05e0-4922-bba0-ea2eb4d7f16e)


We first need to fix the hotkey problem before we can do this task. Which is another issue. I'll create one.

**Comment by ReinerBforartists** [comment]
Just in case, this is the current code for the show/hide class:

```
class VIEW3D_MT_face_sets_showhide(Menu):
    bl_label = "Show/Hide"

    def draw(self, _context):
        layout = self.layout

        layout.operator("sculpt.face_set_change_visibility", text="Toggle Visibility", icon="HIDE_UNSELECTED").mode = 'TOGGLE'

        layout.separator()

        layout.operator("sculpt.face_set_change_visibility", text="Show Active Face Set", icon="HIDE_OFF").mode = 'SHOW_ACTIVE'
        # This here is the operator that Blender uses, so we might want to swap out this one.
        #layout.operator("paint.hide_show_all", text="Show Active Face Set", icon="HIDE_OFF").action = 'SHOW'
        layout.operator("sculpt.face_set_change_visibility", text="Hide Active Face Set", icon="HIDE_ON").mode = 'HIDE_ACTIVE'
```

**Comment by ReinerBforartists** [comment]
Hah, trapped by not recompiled. Hotkeys works. Now let's have a look to reimplement the upper menu ...

**Comment by ReinerBforartists** [comment]
Still some random behaviour, sometimes the non face set geometry hides. And then the show all operator fails. but Blender behaves the same, so...

Documenting

**Comment by ReinerBforartists** [comment]
The issue is solved. But i want to toy around with the operators a bit more and maybe make a bug report to Blender. The current behaviour is everything but ideal. Why do we need to confirm with a mouse click? And why does it sometimes hide everything but the face set?

**Comment by Draise14** [comment]
Click is to "select" the faceset you want to hide/show.

**Comment by ReinerBforartists** [comment]
Ah this explains it. So this is expected behaviour. Hmmm.

**Comment by Draise14** [comment]
Yeah, not much to do. Things are working as expected, I guess we can add the tooltip to mention that.

**Comment by ReinerBforartists** [comment]
Good idea. And fix the manual. I mentioned in the manual it does misbehave ...

**Comment by ReinerBforartists** [comment]
Hmm, to wait for a mouse click might make sense for the hide feature. But once it is hidden, how can you click at hidden geometry to toggle it? O_O

Anyways, let's update the documents ...

**Comment by ReinerBforartists** [comment]
This taks remains a can of worms. One tooltip for all operators. The items in the adjust last operator does have separated tooltips, but not the tool itself.

Separating.

![tooltip](https://github.com/Bforartists/Bforartists/assets/13919702/4d4cdd52-bf70-484c-ac5a-7d1370f156e2)

**Comment by ReinerBforartists** [comment]
Hm, not to separate. I tried all possible and impossible combinations here now. But haven't hit a single working one.

![noway](https://github.com/Bforartists/Bforartists/assets/13919702/8ca03bbc-3bcb-4ff9-92e5-b86661c1296e)

**Comment by ReinerBforartists** [comment]
Did some code cleanup. Let's call it a day, and separate the tooltip separation from this issue.
